model/LtiListenerService.php

```diff
@@ -220,7 +225,7 @@ private function createExecutionContext($executionId, LtiLaunchData $launchData)
                 LtiDeliveryExecutionContext::EXECUTION_CONTEXT_TYPE,
                 $launchData->getVariable(LtiLaunchData::CONTEXT_LABEL)
             );
-        } catch (\InvalidArgumentException $e) {
+        } catch (InvalidArgumentException|LtiVariableMissingException $e) {
```

Comment:
By catching LtiVariableMissingExeception here, the code in [the higher try/catch](https://github.com/oat-sa/extension-lti-proctoring/pull/155/files#diff-b2622eb2310b27a8502e4c68a056c8dd52a4cd87375792f130d92e996dbe6fa9R101) is executed. Do u think it is acceptable?

Reply:
I would say yes. In this case we won't be able to store execution context and probably send some report when proctor terminates the test, but we still will store other data and logs in this handler. In calling method we also catch `LtiVariableMissingExeception`, so the only difference is two lined R99-100. What do you think?